README.md

```diff
@@ -1,75 +1,55 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# STEM - VEX Robotic Conference  and Competition
 
 > Description the project.
+- simple HTML & CSS code, that designs a conference and a robotic competition .
+- Two page website.
+- configure with linters 
 
+# Getting Started
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
+**please get your browser update. To get a local copy up and running follow these simple steps.**
 
-### Setup
+Run the command below from the terminal
 
-### Install
+- git clone in your terminal
+- cd STEM-
+- start index.html
 
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
-## Authors
+## Built With
 
-👤 **Author1**
+- Html & Css
+- JavaScript
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+## Author
 
-👤 **Author2**
+👤 **Abdulai Samuel Moiwa**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/samuelmoiwa)
+- Twitter: [@twitterhandle](https://twitter.com/samuelmoiwa)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ing-abdulai-samuel-moiwa-726340142/)
+ 
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
-
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+Thanks to microverseinc for the linters config files.
+https://github.com/microverseinc/linters-config <br>
 
-## 📝 License
+Thanks to [Cindy Shin](https://www.behance.net/adagio07). <br>
+for the beautiful design guideline that was used in this project.
 
-This project is [MIT](./LICENSE) licensed.
+## [View project live](https://samuelmoiwa.github.io/STEM-/)
+
```

Comment:
[OPTIONAL]
- [ ] I suggest you move the URL to the deployed version to the top of your `README.md` file. It makes it easier for recruiters who only want to view the app and CRs to find it.

Reply:
thanks, :)